### PR TITLE
Update authentication.md

### PR DESCRIPTION
**Pull request by fanxiin**
Fix typo. Change the translation of the word “location” from "为之" to "位置".

### CODE REVIEW

**Comment by tengqm** [comment]
/hold

@fanxiin Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this:

```
./scripts/lsync.sh content/zh/docs/reference/access-authn-authz/authentication.md
```

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -1427,11 +1451,32 @@ users:
...
+
+        MacOS: brew install example-client-go-exec-plugin
+
+        Ubuntu: apt-get install example-client-go-exec-plugin
+
+        Fedora: dnf install example-client-go-exec-plugin
+
+        ...
+
+      # 是否向这个 exec 插件提供集群信息(可能包含非常大的CA数据)，作为KUBERNETES_EXEC_INFO环境变量的一部分。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/authentication.md:1467]:
```suggestion
      # 是否使用 KUBERNETES_EXEC_INFO 环境变量的一部分向这个 exec 插件
      # 提供集群信息（可能包含非常大的 CA 数据）
```

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 2]
```diff
@@ -1427,11 +1451,32 @@ users:
...
 clusters:
 - name: my-cluster
   cluster:
     server: "https://172.17.4.100:6443"
     certificate-authority: "/etc/kubernetes/ca.pem"
+    extensions:
+    - name: client.authentication.k8s.io/exec # 为每个集群 exec 配置保留的扩展名
+      extension:
+        arbitrary: config
+        this: can be provided via the KUBERNETES_EXEC_INFO environment variable upon setting provideClusterInfo
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/authentication.md:1478]:
```suggestion
        this: 在设置 provideClusterInfo 时可通过环境变量 KUBERNETES_EXEC_INFO 指定
```

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 3]
```diff
@@ -1551,3 +1596,34 @@ RFC3339 timestamp. Presence or absence of an expiry has the following impact:
...
+The plugin can optionally be called with an environment variable, `KUBERNETES_EXEC_INFO`,
+that contains information about the cluster for which this plugin is obtaining
+credentials. This information can be used to perform cluster-specific credential
+acquisition logic. In order to enable this behavior, the `provideClusterInfo` field must
+be set on the exec user field in the
+[kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/). Here is an
+example of the aforementioned `KUBERNETES_EXEC_INFO` environment variable.
+-->
+
+该插件可以选择使用环境变量`KUBERNETES_EXEC_INFO`进行调用，
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/authentication.md:1610]:
```suggestion
调用此插件时可以选择性地设置环境变量 `KUBERNETES_EXEC_INFO`。
```

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 4]
```diff
@@ -1551,3 +1596,34 @@ RFC3339 timestamp. Presence or absence of an expiry has the following impact:
...
+该变量包含了此插件获取凭据的集群信息。此信息可用于执行群集特定的凭据获取逻辑。为了启用此行为，必须在 [kubeconfig](/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig/)中的 exec 用户字段上设置`provideClusterInfo`字段。下面是上述`KUBERNETES_EXEC_INFO`环境变量的示例。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/authentication.md:1611]:
```suggestion
该变量包含了此插件获取凭据所针对的集群信息。此信息可用于执行群集特定的凭据获取逻辑。
为了启用此行为，必须在 [kubeconfig](/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
中的 exec user 字段上设置`provideClusterInfo`字段。
下面是上述 `KUBERNETES_EXEC_INFO` 环境变量的示例。
```

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 5]
```diff
@@ -1551,3 +1596,34 @@ RFC3339 timestamp. Presence or absence of an expiry has the following impact:
...
+{
+  "apiVersion": "client.authentication.k8s.io/v1beta1",
+  "kind": "ExecCredential",
+  "spec": {
+    "cluster": {
+      "server": "https://172.17.4.100:6443",
+      "certificate-authority-data": "LS0t...",
+      "config": {
+        "arbitrary": "config",
+        "this": "can be provided via the KUBERNETES_EXEC_INFO environment variable upon setting provideClusterInfo",
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/authentication.md:1623]:
```suggestion
        "this": "在设置 provideClusterInfo 时可通过环境变量 KUBERNETES_EXEC_INFO 指定",
```

**Comment by fanxiin** [comment]
@tengqm Thanks for your review!

**Comment by tengqm** [comment]
Thanks.
/approve

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**